tests/test_colorinterp.py

```diff
@@ -24,9 +24,7 @@ def test_cmyk_interp(tmpdir):
             ColorInterp.black)
 
 
-@pytest.mark.skipif(
-    Version(rasterio.__gdal_version__) < Version('2.2.2'),
-    reason="Some prior versions segfault on a Mac OSX Homebrew GDAL install.")
+@requires_gdal22  # Some versions prior to 2.2.2 segfault on a Mac OSX Homebrew GDAL
```

Comment:
Seems like we could maybe move that comment into the decorator?

```
@requires_gdal22(reason="Some versions prior to 2.2.2 segfault on a Mac OSX Homebrew GDAL")
```

Reply:
Yes, we can.  Do you want this for just specific comments, or everywhere that I moved `reason` into a comment?